api/HTMLIsIndexElement.json

```diff
@@ -1,148 +0,0 @@
-{
-  "api": {
-    "HTMLIsIndexElement": {
```

Comment:
I think this should be paired with removal of https://github.com/mdn/browser-compat-data/blob/master/html/elements/isindex.json.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/isindex says "All major browsers have now removed `<isindex>`" but not when.

Per https://github.com/whatwg/html/pull/1095 support was removed from Edge and Chrome back then, and Firefox was planning to remove. That happened here:
https://github.com/mozilla/gecko-dev/commit/217ac88f04911b1c3c956f7b70f431735ab244ae

WebKit support was dropped here:
https://trac.webkit.org/changeset/205858/webkit

So yes, this is far enough in the past that we can drop it from MDN/BCD.

Reply:
It's covered in https://github.com/mdn/browser-compat-data/pull/6955 but I can also do it here. Do you prefer to do it here?

Reply:
Nah, let's leave it in the other PR, I didn't know it was covered there.